src/semgrep_agent/semgrep.py

```diff
@@ -128,21 +128,23 @@ def invoke_semgrep(
 
     debug_echo("=== seeing if there are any findings")
     findings = FindingSets()
-    rules_path_len = (
-        len(os.path.dirname(config_specifier)) if uses_managed_policy else 0
-    )
 
     with targets.current_paths() as paths:
         click.echo(
             "=== looking for current issues in " + unit_len(paths, "file"), err=True
         )
         for chunk in chunked_iter(paths, PATHS_CHUNK_SIZE):
-            args = ["--skip-unknown-extensions", "--json", *config_args]
+            args = [
+                "--skip-unknown-extensions",
+                "--json",
+                "--no-rewrite-rule-ids",
+                *config_args,
+            ]
```

Comment:
if uses_managed policy else (do not include the --no-rewrite-rule-ids) per Bence's commment on slack